src/main/resources/swagger/api-docs.yaml

```diff
@@ -6876,6 +6876,107 @@ paths:
           description: Internal Server Error
           content: {}
       x-passthrough: false
```

Comment:
These swagger changes are copy-pasted from Rawls, then tweaked just slightly to be compatible with orch, e.g.:
* rename Rawls' `workspaceNamespacePathParam` to Orch's `workspaceNamespaceParam`
* use the `Entities` tag instead of Rawls' `entities`